PWGHF/TableProducer/candidateSelectorDstarToD0Pi.cxx

```diff
@@ -187,7 +187,7 @@ struct HfCandidateSelectorDstarToD0Pi {
     if (std::abs(candidate.impParamSoftPi()) > cutsDstar->get(binPt, "d0SoftPi")) {
       return false;
     }
-    if (std::abs(candidate.normalisedImpParamSoftPi()) > cutsDstar->get(binPt, "d0SoftPiNormalised")) {
+    if (std::abs(candidate.normalisedImpParamSoftPi()) < cutsDstar->get(binPt, "d0SoftPiNormalised")) {
```

Comment:
This was correct as it was, since the soft pion is not displaced (it comes from the primary vertex, at least for prompt production). Hence, if the value is larger than a given value it should be rejected (i.e. it should return `false`)

Reply:
OK. So what should be that threshold value?  In train run on hyperloop I used value 0.5.

Reply:
You can put a very large value for the moment (i.e. 1000) so that it does not cut any candidate, and it can be then studied in the future